### PR TITLE
feat(storagecontrol): Folders samples

### PR DESCRIPTION
**Pull request by JesseLovelace**
Adds samples for the Folders feature in storage control

### CODE REVIEW

**Review by amanda-tarafa — APPROVED** [review]

**Comment by amanda-tarafa** [comment]
Looks good, but you have some `myFolder` left still, see CI error logs:

```
Failed Storage.Samples.Tests.StorageControlGetFolderTest.TestStorageControlGetFolder [321 ms]
  Error Message:
   Assert.Contains() Failure: Sub-string not found
String:    "projects/_/buckets/8672defb-e8d6-4d86-a3b"···
Not found: "myFolder"
  Stack Trace:
     at Storage.Samples.Tests.StorageControlGetFolderTest.TestStorageControlGetFolder() in /tmpfs/src/github/dotnet-docs-samples/storage/api/Storage.Samples.Tests/StorageControlGetFolderTest.cs:line 38
```

File: storage/api/Storage.Samples.Tests/StorageControlRenameFolderTest.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    {
+        _fixture = fixture;
+    }
+
+    [Fact]
+    public void TestStorageControlRenameFolder()
+    {
+        StorageControlCreateFolderSample createSample = new StorageControlCreateFolderSample();
+        var folder = createSample.StorageControlCreateFolder(_fixture.BucketNameHns, "renameTestFolder");
+
```
Comment (amanda-tarafa) [inline storage/api/Storage.Samples.Tests/StorageControlRenameFolderTest.cs:34]: I think you might want to call `_fxiture.SleepAfterBucketCreation()` after each bucket creation, here and everywhere.

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
Just one comment, to avoid hitting quotas.

File: storage/api/Storage.Samples.Tests/StorageControlRenameFolderTest.cs (hunk 1 above)
Comment (JesseLovelace) [reply storage/api/Storage.Samples.Tests/StorageControlRenameFolderTest.cs:34]: There is a sleep after the bucket is created in StorageFIxture--this is just creating a Folder within the bucket, it doesn't create a new bucket

Comment (amanda-tarafa) [reply storage/api/Storage.Samples.Tests/StorageControlRenameFolderTest.cs:34]: And is not subject to the same quotas then?

**Review by amanda-tarafa — APPROVED** [review]
Thanks!

File: storage/api/Storage.Samples.Tests/StorageControlRenameFolderTest.cs (hunk 1 above)
Comment (JesseLovelace) [reply storage/api/Storage.Samples.Tests/StorageControlRenameFolderTest.cs:34]: Correct, not subject to those quotas

**Pull request merged**